### ISSUE TITLE
Parsing time failed

### ISSUE THREAD

**Issue by AlexKovalevych** [kennellroxco/xlsxir#42]
```
** (FunctionClauseError) no function clause matching in Xlsxir.ConvertTime.from_float/1
    lib/xlsxir/convert_time.ex:31: Xlsxir.ConvertTime.from_float(42705.0002662037)
    lib/xlsxir/parse_worksheet.ex:59: Xlsxir.ParseWorksheet.sax_event_handler/2
    src/erlsom_sax_utf8.erl:1408: :erlsom_sax_utf8.wrapCallback/2
    src/erlsom_sax_utf8.erl:907: :erlsom_sax_utf8.parseContentLT/2
    src/erlsom_sax_utf8.erl:196: :erlsom_sax_utf8.parse/2
    lib/xlsxir/sax_parser.ex:59: Xlsxir.SaxParser.parse/2
    lib/xlsxir.ex:137: Xlsxir.do_multi_extract/3
    lib/xlsxir.ex:109: anonymous fn/4 in Xlsxir.multi_extract/3
```

**Comment by AlexKovalevych** [comment]
I guess we can use this function https://github.com/AlexKovalevych/exoffice/blob/master/lib/exoffice/parser/excel_2003/date.ex 
for convertion (its for excel 2003 format, but still should work)

**Comment by AlexKovalevych** [comment]
Something wrong with condition which defines whether its time or date, will investigate.

**Comment by AlexKovalevych** [comment]
#43

**Comment by kennellroxco** [comment]
This issue was fixed by PR #45.
